docs/generating-a-provider.md

```diff
@@ -129,28 +127,24 @@ be quite similar for any other Terraform provider.
    ```
 
    ```bash
+   # Note that you need to change `myorg/provider-github`.
    cat <<EOF > config/branch/config.go
    package branch
 
    import "github.com/upbound/upjet/pkg/config"
 
    func Configure(p *config.Provider) {
        p.AddResourceConfigurator("github_branch", func(r *config.Resource) {
-
-           // we need to override the default group that upjet generated for
+           // We need to override the default group that upjet generated for
            // this resource, which would be "github"
            r.ShortGroup = "branch"
 
-           // Identifier for this resource is assigned by the provider. In other
-           // words it is not simply the name of the resource.
-           r.ExternalName = config.IdentifierFromProvider
-
            // This resource need the repository in which branch would be created
            // as an input. And by defining it as a reference to Repository
            // object, we can build cross resource referencing. See
            // repositoryRef in the example in the Testing section below.
            r.References["repository"] = config.Reference{
-               Type: "github.com/crossplane-contrib/provider-jet-github/apis/repository/v1alpha1.Repository",
+               Type: "github.com/myorg/provider-github/apis/repository/v1alpha1.Repository",
```

Comment:
Looking at the examples below, should `myorg` be `upbound` or vice versa?

Reply:
It should be `myorg` everywhere in the user's provider since they'll need to use their own organization. Is there an inconsistency you caught?